src/main/java/uk/gov/justice/hmpps/prison/service/CaseNoteService.java

```diff
@@ -100,7 +102,27 @@ public CaseNoteService(final CaseNoteRepository caseNoteRepository,
         this.staffUserAccountRepository = staffUserAccountRepository;
     }
 
-    public Page<CaseNote> getCaseNotes(final CaseNoteFilter caseNoteFilter, final Pageable pageable) {
+    public Page<CaseNote> getCaseNotes(final String offenderNo,final LocalDate from,final LocalDate to,final String type,final String subType,final String prisonId,final List<String> caseNoteTypeSubTypes, final Pageable pageable) {
+        final var latestBookingByOffenderNo = bookingService.getLatestBookingByOffenderNo(offenderNo);
+        final var types = new ArrayList<>(Optional.ofNullable(caseNoteTypeSubTypes).orElseGet(Collections::emptyList));
+        //Add request param type and subtype for backward compatibility
+        if(type!=null){
```

Comment:
see comment above - this can be moved to the builder which can validate the parameters and construct the filter